src/storage/servers.js

```diff
@@ -0,0 +1,85 @@
+const db = require('../storage/DataAccess');
+const { connectToServer } = require('../utils/client');
+
+const SERVERS = 'serverConfig';
+
+/**
+ * Fields for Servers Data Types:
+ *
+ * Server:
+ *  @param {string} id - the id of the server
+ *  @param {string} name - human readable name
+ *  @param {string} endpoint: - FHIR base url
+ *  @param {string} type - 'KA' | 'EHR' | 'PHA' - type of server the endpoint is
+ *  @param {string} clientId - the client ID assigned to this app during registration
+ *  @param {string} token - the last access token received from authorization
+ *  @param {number} tokenExp - the time the token expires (UTC)
+ *  @param {string} key - the last key
+ *
+ */
+
+class Servers {
```

Comment:
I'm also not convinced this needs to be a class. Since there is no `this.server` attribute and each function takes in the `server` as a parameter

Reply:
This is no longer a class